### PR TITLE
Chore(lua-language-server): update to 2.5.6

### PR DESCRIPTION
**Pull request by MrAdityaAlok**
- remove arm from TERMUX_PKG_BLACKLISTED_ARCHES

Signed-off-by: Aditya Alok <dev.aditya.alok@gmail.com>

#8491 should be closed. That's a wrong patch.

### CODE REVIEW

**Review by thunder-coding — APPROVED** [review]

File: packages/lua-language-server/build.sh [hunk 1]
```diff
@@ -2,18 +2,16 @@ TERMUX_PKG_HOMEPAGE="https://github.com/sumneko/lua-language-server"
...
 TERMUX_PKG_GIT_BRANCH="${TERMUX_PKG_VERSION}"
+TERMUX_PKG_SRCURL="https://github.com/sumneko/lua-language-server.git"
 TERMUX_PKG_BUILD_DEPENDS="libandroid-spawn"
 TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
-# no cpu_relax support present for these archs.
-# https://github.com/actboy168/bee.lua/blob/32f65b92739fa236d87fc1b2e7617470d47f0355/bee/thread/spinlock.h#L14
-TERMUX_PKG_BLACKLISTED_ARCHES="arm,i686"
+TERMUX_PKG_BLACKLISTED_ARCHES="i686"
```
Comment (shadmansaleh) [inline packages/lua-language-server/build.sh:12]: Can you try removing `i686` too it's build succeeded for me too. https://github.com/shadmansaleh/termux-packages/pull/18

Comment (thunder-coding) [reply packages/lua-language-server/build.sh:12]: If you can create a PR here doing the same. I'd be happy to merge that.

File: packages/lua-language-server/build.sh [hunk 2]
```diff
@@ -69,9 +67,9 @@ termux_step_make_install() {
 
 	chmod 744 "${TERMUX_PREFIX}/bin/${TERMUX_PKG_NAME}"
 
-	install -Dm744 -t "${INSTALL_DIR}"/bin/Android ./bin/Android/"${TERMUX_PKG_NAME}"
+	install -Dm744 -t "${INSTALL_DIR}"/bin ./bin/"${TERMUX_PKG_NAME}"
 	install -Dm644 -t "${INSTALL_DIR}" ./{main,debugger}.lua
-	install -Dm644 -t "${INSTALL_DIR}"/bin/Android ./bin/Android/main.lua
+	install -Dm644 -t "${INSTALL_DIR}"/bin ./bin/main.lua
```
Comment (shadmansaleh) [inline packages/lua-language-server/build.sh:72]: Maybe this should go to `share/` or somewhere similar ? `bin/` usually contains binaries ?

File: packages/lua-language-server/build.sh (hunk 1 above)
Comment (shadmansaleh) [reply packages/lua-language-server/build.sh:12]: Sure :)

File: packages/lua-language-server/build.sh (hunk 2 above)
Comment (thunder-coding) [reply packages/lua-language-server/build.sh:72]: Nah, consider checking out `INSTALL_DIR` environment variable. Also Arch is installing the file at that location only https://archlinux.org/packages/community/x86_64/lua-language-server/

:)

Comment (shadmansaleh) [reply packages/lua-language-server/build.sh:72]: No @thunder-coding  the arch package is installing the wrapper script in `bin/` and installing the `lua-language-server` binary and the `main.lua` file under `lib/` . See [here](https://github.com/archlinux/svntogit-community/blob/788278b1dca83ec6bcccaeeae6d8b06b758d25e1/trunk/PKGBUILD#L52-L57)

**Comment by MrAdityaAlok** [comment]
@shadmansaleh look here https://github.com/termux/termux-packages/blob/93102bbf13cd7b7a36b982516da93b327120fbd5/packages/lua-language-server/build.sh#L49 Wrapper is installed in "$PREFIX/bin"

**Comment by shadmansaleh** [comment]
I wasn't talking about the wrapper . I was talking about `main.lua`

**Pull request merged**